src/ontodia/widgets/node.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import { FatClassModel } from '../diagram/elements';
+import { formatLocalizedLabel } from '../diagram/model';
+import { TreeNodes } from './treeNodes';
+
+export interface NodeTreeProps {
+    node: FatClassModel;
+    lang?: Readonly<string> | undefined;
```

Comment:
`string` is already immutable, so it doesn't make sense to wrap it as `Readonly<string>`

Reply:
Ofc. Corrected.